packages/account/src/Components/api-token/__tests__/api-token-article.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import ApiTokenArticle from '../api-token-article';
+
+it('should render ApiTokenArticle', () => {
+    render(<ApiTokenArticle />);
+    expect(screen.getByText('API token'));
```

Comment:
expect what? 🤔 
Maybe you need to add sth like this
```suggestion
    expect(screen.getByText('API token')).toBeInTheDocument();
```